fix(css): clarify unitless zero for angle values (#44147)
mdn/content: files/en-us/web/css/reference/values/angle/index.md

## Syntax

> The `<angle>` data type consists of a {{cssxref("&lt;number&gt;")}} followed by one of the units listed below. As with all dimensions, there is no space between the unit literal and the number. Some legacy uses of `<angle>` allow a unitless zero angle, but this is not generally true. To represent a zero angle, include a unit such as `0deg`, `0grad`, `0rad`, or `0turn`.

Reviewer: Slightly more succinct 

```suggestion
The `<angle>` data type consists of a {{cssxref("&lt;number&gt;")}} followed by one of the units listed below. As with all dimensions, there is no space between the unit literal and the number.
While some legacy uses accept a unitless `0`, you should always include a unit for zero values, such as: `0deg`, `0grad`, `0rad`, or `0turn`.
```
